config/fxdata/terrain.cfg

```diff
@@ -1080,11 +1079,11 @@ Cost = 0
 Health = 0
 SlabAssign = HARD
 Roles = 
-; HAS_NO_ENSIGN CANNOT_VANDALIZE BUILD_TILL_BROKE CANNOT_BE_SOLD 
+; BUILD_TILL_BROKE - CANNOT_BE_CLAIMED - CANNOT_BE_SOLD - CANNOT_VANDALIZE - HAS_NO_ENSIGN.
```

Comment:
I do not think it is smart to document it like this. People might think they need the ` - ` in the properties if they want multiple.